ci/pipeline.yml

```diff
@@ -95,6 +95,20 @@ jobs:
       AWS_DEFAULT_REGION: {{aws-region}}
       GATEWAY_HOST: {{gateway-host}}
 
+- name: prometheus-down-check
+  serial_groups: [production]
+  plan:
+  - aggregate:
+    - get: prometheus-check-timer
+      trigger: true
+    - get: prometheus-config
+      resource: prometheus-config
```

Comment:
The `resource` directive is used to alias the resource named in `resource` to the label named in `get`. Since the label and the resource name are the same here, the `resource` directive doesn't do anything.

Reply:
Again, cut/paste.  Do we even need that resource?  I was going to ask today if it was even needed.

Reply:
I guess we do, because it supplies the scripts.  So I'm confused as to what we need to do here, then.  Can you clarify what you think should be done here?

Reply:
what @jmcarp is saying is that since you are doing a `get: prometheus-config` and it is the name of a resource, you don't have to actually define the `resource`.